### ISSUE TITLE
Newest version, Error rcvd: NULL Problem importing Power CAT component library

### ISSUE THREAD

**Issue by CCampbell2074** [microsoft/powercat-creator-kit#331]
I just imported the newest version of creator kit and the associated Canvas and MDA solutions. Now when I open an app, click the 'Review' button to update the components used, all controls disappear and I get the following error. 

![PowerCATerror](https://github.com/microsoft/powercat-creator-kit/assets/101434649/4e4e3f54-703a-46f6-abf6-7c9fe2a18537)

**Comment by CCampbell2074** [comment]
Hello. I haven’t installed the latest release yet. I’ll test that out today
and let you know.

**Comment by CCampbell2074** [comment]
> @CCampbell2074 are you still seeing this error with the latest release?

Yes. This version is the one that has issues and the components no longer work. I didn't have any issues with the previous version.

**Comment by CCampbell2074** [comment]
These are the components I'm using in my canvas app. See screenshot. 
![image](https://github.com/microsoft/powercat-creator-kit/assets/101434649/70bae442-337e-4b04-ab59-12dbea0a8aae)

**Comment by PEP-Dustin** [comment]
I'm having this same issue in the latest release (2024).  I tried prior release (2023) and also get the same error, as well as when installing the AI kit.  When I try to use the PowerCAT components after installing, I only see 5 core components (Dialog, Header, Material, Panel, and ExpandMenu).  None of the AI components become available in my apps.
